src/components/AllReviews.jsx

```diff
@@ -0,0 +1,54 @@
+import { useEffect, useState } from "react";
+import { fetchReviews } from "../api";
+
+const AllReview = () => {
+  const [allReviews, setAllReviews] = useState([]);
+  const [page, setPage] = useState("1");
+  const [next, setNext] = useState("");
+  const [previous, setPrevious] = useState("");
+  useEffect(() => {
+    fetchReviews(page).then((data) => {
+      setAllReviews(data.results);
+      setNext(data.next.page);
+      setPrevious(data.previous.page);
```

Comment:
On the first page, there is no data.previous that you have access to. You will need to find a way around this, as currently it's throwing an error.